.gitignore

```diff
@@ -26,6 +26,10 @@ coverage
 # Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
 .grunt
 
+# Credentials
```

Comment:
I'm not 100% sure on this, but I think you might be able to ignore both of these with just credentials.js (so that any file in any directory with this name will be ignored)

Reply:
That worked.  Thanks.